tests/acceptance/fixtures.py

```diff
@@ -131,6 +131,8 @@ def qemu_finalizer():
         def qemu_finalizer_impl(conn):
             try:
                 manual_uboot_commit(conn)
+                # Collect the coverage files from /data/mender/ if present
+                get_no_sftp("/data/mender/cover*", conn, local="coverage")
```

Comment:
If there are no files, will this throw, or just silently fail? It's a cleanup function, so important that the rest also runs.

Reply:
To be honest, I'm not sure yet, but it's on the roadmap to figure it out ;)